app/controllers/targets_controller.rb

```diff
@@ -38,13 +44,20 @@ def load_create_target
     @target = Target.new
     render json: { form: (render_to_string partial: 'create_target') }
   end
+
+  def load_delete_target
+    target_id = params[:target_id]
+    @target = Target.find(target_id)
+    render json: { target: (render_to_string partial: 'delete_target') }
```

Comment:
it's a good practice passing the variables to the partial as locales instead of using instance variables.